### ISSUE TITLE
Slick adding slick-cloned class in the new slides.

### ISSUE THREAD

**Issue by arnab-bhattacharjee-codeclouds** [gs-shop/vue-slick-carousel#133]
My Carousel HTML
----------------------
<VueSlickCarousel :arrows="true" :dots="false" v-bind="carousel_settings" ref="carousel" @init="onInitCarousel" @reInit="onInitCarousel">

My Carousel Setting
-----------------------
carousel_settings: {
	centerMode: false,
	centerPadding: "20px",
	focusOnSelect: false,
	infinite: false,
	slidesToShow: 1,
	speed: 500,
	responsive: [],
	adaptiveHeight: false,
}

I am using this package for 3 months, but recently I am having one issue regarding the slick-cloned class. At first, I am rendering 10 slides after getting the response from server-side. But after click next, I am adding 10 more elements in the array. It was working perfectly but for a few days I am getting an issue of the cloned and the next button is getting disabled.

<img width="583" alt="Screenshot 2020-08-19 at 7 58 40 PM" src="https://user-images.githubusercontent.com/59700656/90648036-ad40ee80-e256-11ea-9dd7-635620b62940.png">

**Comment by yiyasha15** [comment]
@arnab-bhattacharjee-codeclouds were you able to fix this?

**Comment by arnab-bhattacharjee-codeclouds** [comment]
> 

No, I wasn't able to fix this issue. So I changed the slick slider to a swiper slider.

**Comment by yiyasha15** [comment]
Ah I see actually I just added a prop :centerMode="true" and worked for me.. in case you wanna try.

**Comment by arnab-bhattacharjee-codeclouds** [comment]
> Ah I see actually I just added a prop :centerMode="true" and worked for me.. in case you wanna try.

Not required now, it's been 3 years and the project is running successfully.
